src/signature_verifier.rs

```diff
@@ -140,10 +141,9 @@ impl Error for SlackEventWrongSignatureError {}
 
 #[test]
 fn check_signature_success() {
-    let rng = ring::rand::SystemRandom::new();
-    let key_value: [u8; ring::digest::SHA256_OUTPUT_LEN] =
-        ring::rand::generate(&rng).unwrap().expose();
-    let key_str: String = hex::encode(key_value);
+    use sha2::Digest;
+
+    let key_str: String = hex::encode(Sha256::digest("test-key"));
```

Comment:
Nit: Preferrable to generate keys all the time new, so we don't have specific behaviour for specific keys yet failing for others.